cmd/kube-controller-manager/app/certificates.go

```diff
@@ -167,6 +171,7 @@ func startCSRCleanerController(ctx context.Context, controllerContext Controller
 }
 
 func startRootCACertPublisher(ctx context.Context, controllerContext ControllerContext) (controller.Interface, bool, error) {
+	ctx = klog.NewContext(ctx, klog.LoggerWithName(klog.FromContext(ctx), "root CA publisher"))
```

Comment:
for this one it should be `root-ca-cert-publisher-controller`

 https://github.com/kubernetes/kubernetes/blob/6ce055d62d8608842db18901c948c6982c74a164/cmd/kube-controller-manager/app/controllermanager.go#L472